apps/chatbots/views.py

```diff
@@ -307,9 +311,108 @@ def archive_chatbot(request, team_slug: str, pk: int):
     return HttpResponseClientRedirect(reverse("chatbots:chatbots_home", kwargs={"team_slug": team_slug}))
 
 
-class CreateChatbotVersion(CreateExperimentVersion):
-    permission_required = "experiments.add_experiment"
+class CreateChatbotVersion(LoginAndTeamRequiredMixin, PermissionRequiredMixin, TemplateView):
```

Comment:
The previous class extended `FormView` but this extends `TemplateView`. Why the change?